boards/arm/lpcxpresso54114_m0/doc/lpcxpresso54114.rst

```diff
@@ -0,0 +1,61 @@
+.. _lpcxpresso54114_m0:
+
+NXP LPCXPRESSO54114
```

Comment:
might-should add ``M0`` or maybe ``(M0 core)`` to this title.  Without it, it has the same title as the board doc we've already got.